frontend/src/components/PageFooter.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
```

Comment:
Why encapsulate footer in div and not just put in footer directly?

Reply:
That's a VueJS's requirement.